### PR TITLE
ci: add job to set the milestone of linked issues to match the milestone of the PR [skip ci]

### PR DESCRIPTION
**Pull request by cpcloud**
Add an action to automatically set the milestone on linked issues after they are resolved by a PR.

### CODE REVIEW

File: ci/set-milestone-on-issue.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+top="$(dirname "$(readlink -f "$0")")"
+
+# find all pull requests associated with commit
+jq='.data.repository.pullRequest.closingIssuesReferences.nodes[].number'
```
Comment (gforsyth) [inline ci/set-milestone-on-issue.sh:8]: can we use a different variable name?

**Review by gforsyth — APPROVED** [review]
looks good to me -- minor nit on a variable name, but non-blocking

File: ci/set-milestone-on-issue.sh (hunk 1 above)
Comment (cpcloud) [reply ci/set-milestone-on-issue.sh]: changed to `linked_issues_query`

**Comment by cpcloud** [comment]
Sweet, looks like it worked!

image

**Pull request merged**